### DISCUSSION TITLE
Where can I find working links mentioned in documentation?

### DISCUSSION THREAD

**Discussion by KaushikEDS** [IgniteUI/igniteui-angular#13959] [Q&A] [answered]
The links on this page show 404
https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/grid 

For example :-
https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/igxcolumncomponent.html
https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/igxgridcomponent.html#autoGenerate

![image](https://github.com/IgniteUI/igniteui-angular/assets/79090369/1bd54132-43cb-4717-97e2-23a1baab0bb5)
![image](https://github.com/IgniteUI/igniteui-angular/assets/79090369/db557298-040a-4783-a613-2ec184cfd819)

**Comment by kacheshmarova** [comment]
@KaushikEDS This is a known issue on our side that we are currently working on. I will let you know as soon as it is fixed.

**Comment by kacheshmarova** [comment, marked as answer]
@KaushikEDS This issue is fixed and all links are working as expected.
